app/main.py

```diff
@@ -1 +1,23 @@
-# write your code here
+from app.players.player import Player
+from app.players.dwarves.dwarf import Dwarf
+from app.players.dwarves.dwarf_warrior import DwarfWarrior
+from app.players.dwarves.dwarf_blacksmith import DwarfBlacksmith
+from app.players.elves.elf import Elf
+from app.players.elves.druid import Druid
+from app.players.elves.elf_ranger import ElfRanger
+
+
+def calculate_team_total_rating(players: list[Player]) -> int:
+    return sum(player.get_rating() for player in players)
+
+
+def elves_concert(elves: list[Elf, Druid, ElfRanger]) -> None:
```

Comment:
```suggestion
def elves_concert(elves: list[Elf]) -> None:
```
As Druid, ElfRanger are also elves